### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by oieraraiz** [johnkeel-thork/TET#5] [Translations]
How can I help you with the translation?
I don´t understand how the system works. 
I think that I can help you in french, Spanish and even in Basque

**Comment by ndias666** [comment]
I could help with Portuguese (mother tongue).

**Comment by ndias666** [comment]
@johnkeel-thork do we need new translations? Why don't we use the texts from Android app?
We create the files for iOS, copy the texts from Android to iOS folders.

**Reply by johnkeel-thork** [reply]
We don't need new complete translations, I think there might be some countries missing and the next Android app update will also need new Strings to be translated.
As the app progresses, both on Android and iOS there will always be a need for updating all translations.

**Reply by ArnalOMallet** [reply]
Hi guy. I found some errors in french translation and some strings to translate. I did it locally but cant push cause no write access. If you want it dont hesitate ;)

**Reply by johnkeel-thork** [reply]
Hello, you usually just do a merge request and we will approve it.
If you want you can email it to me: info@drivemodedashboard.com or just send me a link for your repository with the translated file.
